staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -287,6 +322,8 @@ func (o *AuditLogOptions) AddFlags(fs *pflag.FlagSet) {
 		"Format of saved audits. \"legacy\" indicates 1-line text format for each event."+
 			" \"json\" indicates structured json format. Requires the 'AdvancedAuditing' feature"+
 			" gate. Known formats are "+strings.Join(pluginlog.AllowedFormats, ",")+".")
+	fs.StringVar(&o.GroupVersionString, "audit-log-version", o.GroupVersionString,
```

Comment:
audit-log-group-version

Reply:
I'd keep it short `audit-log-version`. Everyone familiar with k8s will expect version to be group + actual version